ansible/roles/stack-sunbird/templates/sunbird_lms-service.env

```diff
@@ -127,3 +127,12 @@ learning.content.props.to.add={{ learning_content_props_to_add | default('mimeTy
 group_activity_agg_cache_ttl={{ group_activity_agg_cache_ttl }}
 group_activity_agg_cache_enable={{ group_activity_agg_cache_enable }}
 sunbird_env_name={{env_name}}
+
+## Redis connection detail ##
+redis.connection.max={{ redis_connection_max | default(64) }}
```

Comment:
varialbe should have sevice name prefix, like `lms_redis_`

Reply:
Changes dones